sql/cw_requests.sql

```diff
@@ -12,4 +12,5 @@ CREATE TABLE /*_*/cw_requests (
   cw_category VARCHAR(64) NOT NULL,
   cw_visibility TINYINT UNSIGNED NOT NULL DEFAULT '0',
   cw_bio TINYINT UNSIGNED NOT NULL DEFAULT '0'
+  cw_lock TINYINT NOT NULL DEFAULT '0'
```

Comment:
Doesn't match the SQL patch.
```suggestion
  cw_lock TINYINT UNSIGNED NOT NULL DEFAULT '0'
```